CHANGELOG.md

```diff
@@ -1,3 +1,36 @@
+## [15.1.4](https://github.com/commercetools/merchant-center-application-kit/compare/v15.1.3...v15.1.4) (2019-09-11)
+
+#### 🐛 Type: Bug
+
+- `application-shell-connectors`, `permissions`
+  - [#1049](https://github.com/commercetools/merchant-center-application-kit/pull/1049) fix: to avoid using Mapped Types as babel plugin does not properly generate prop-types ([@emmenko](https://github.com/emmenko))
+- `mc-html-template`
+  - [#1045](https://github.com/commercetools/merchant-center-application-kit/pull/1045) fix(csp): to whitelist sentry.io in connect-src ([@emmenko](https://github.com/emmenko))
+
+#### 🔮 Type: Chore
+
+- `i18n`, `permissions`
+  - [#1047](https://github.com/commercetools/merchant-center-application-kit/pull/1047) chore: improve ts setup for discovering types ([@emmenko](https://github.com/emmenko))
+
+#### 💅 Type: Enhancement
+
+- `application-shell-connectors`, `application-shell`
+  - [#1048](https://github.com/commercetools/merchant-center-application-kit/pull/1048) feat: validate graphql files using eslint and schema introspection ([@emmenko](https://github.com/emmenko))
+
+#### ⛑ Type: Refactoring
+
+- `application-shell`
+  - [#1038](https://github.com/commercetools/merchant-center-application-kit/pull/1038) refactor(app-shell): do not fetch unused fields for project query ([@emmenko](https://github.com/emmenko))
+
+#### 🤖 Type: Dependencies
+
+- `eslint-config-mc-app`, `jest-preset-mc-app`, `mc-html-template`, `mc-scripts`, `sentry`
+  - [#1040](https://github.com/commercetools/merchant-center-application-kit/pull/1040) chore(deps): update all dependencies (major) ([@renovate[bot]](https://github.com/apps/renovate))
+- `actions-global`, `application-components`, `application-shell-connectors`, `application-shell`, `babel-preset-mc-app`, `browser-history`, `eslint-plugin-testing-library-react`, `i18n`, `jest-preset-mc-app`, `jest-stylelint-runner`, `l10n`, `mc-html-template`, `mc-scripts`, `notifications`, `permissions`, `react-notifications`, `sentry`, `url-utils`
+  - [#1039](https://github.com/commercetools/merchant-center-application-kit/pull/1039) fix(deps): update all dependencies ([@renovate[bot]](https://github.com/apps/renovate))
```

Comment:
As mentioned on the PR. The update of Apollo to v3.1 could break views in the MC. The `data` prop now maybe `undefined` instead of `{}` incase the query is loading an error occured etc.

Reply:
okay.
after a chat with Tobi, what we'll do is...

1. release it 
2. open a PR on MC-FE to update app-kit, deploy a branch and urge teams to test/fix their views